### ISSUE TITLE
Support real-time streaming channels

### ISSUE THREAD

**Issue by dgoulet** [d-e-s-o/apca#60]
Listed here are all the real-time data channels that Alpaca provides:

https://alpaca.markets/docs/api-references/market-data-api/stock-pricing-data/realtime/#subscribe

At the moment, `apca` is missing:

- dailyBars
- updatedBars
- statuses
- lulds

Would be great to add support for them. At least the `dailyBars` and `updatedBars` seems quite trivial to add as they are streaming an already implemented interface (`Bar`).

**Comment by d-e-s-o** [comment]
Makes sense. I currently have no use for them. If you have an implementation feel free to propose it for inclusion.

**Comment by jaredmcqueen** [comment]
hey @dgoulet I wrote my own library to provide those:
https://github.com/jaredmcqueen/mini-alpaca
